docker/irods_client/tests/test_session_cached_pw.py

```diff
@@ -0,0 +1,17 @@
+import os
+from ibridges import Session
+
+def test_session_from_cached_pw(config, irods_env):
+    # test only for plain irods
+    if not "set_home_perm" in config:
+        session = Session(irods_env_path=os.path.expanduser("~/.irods/irods_environment.json"))
+        assert session.has_valid_irods_session()
```

Comment:
If we're moving it to test_session.py, let's also create a function that does these checks so we have less copy-pasting.

Reply:
We can also add an entry `"environment": "yoda"` and `"environment": "irods4.3"` to the respective configs. THat would make the checks less prone to bugs.

Reply:
You'll have to explain this a bit more. For what purpose do you want the environment name in there? Which checks would be less prone to bugs?

Reply:
I added more flags in the config.toml which we can use for checking whether we want to test a certain feature in the respective environment. E.g. I added "test_user"="tesuser", a second user in our plain iRODS setup for the ACL tests in docker/irods_client/environments/plain-irods/config.toml
And I added env_path = "/root/.irods/irods_environment.json" for the plain irods, while yoda still uses the previous location of the file ibridges/integration_test/environment.

I was indeed not very clear in my statement above. I do not mean bugs, but in our tests we rely on certain ways that the iRODS server and the client docker are set up which differ and hence make the unit tests fail. Not because of real bugs, but simply because of configuration differences of the two environments.
I used the config files to introduce flags in the test code.

Reply:
Yes, I think that's the right way to do it.